webapp/tests/Unit/Controller/PublicControllerTest.php

```diff
@@ -95,6 +96,19 @@ public function testSelfRegister(array $inputs, string $password, array $fixture
                 self::assertSelectorExists('html:contains("'.$inputs[$field].'")');
             }
         }
+
+        /** @var User $user */
+        $user = static::getContainer()
+            ->get(EntityManagerInterface::class)
+            ->getRepository(User::class)
+            ->findOneBy(['username' => $inputs['username']]);
+
+        self::assertNotNull($user);
+        self::assertNotEmpty($user->getExternalid());
+        self::assertNotEmpty($user->getTeam()->getExternalid());
```

Comment:
Should the category also be checked?

Reply:
The category is not created during register but by an admin. So that already has an external id